### PR TITLE
Create Record with InternalApi

### PR DESCRIPTION
**Pull request by philippgualdi**
**DEMO**

Creating fake records with invenio:

`$ invenio marc21 demo `

Added some tests creating example records with the Marc21RecordService

### CODE REVIEW

File: invenio_records_marc21/api.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+from . import models
+
+
+class Marc21Draft(Draft):
+    """Marc21 draft API."""
+
+    model_cls = models.DraftMetadata
+
+    index = IndexField("marc21-drafts-marc21-v1.0.0", search_alias="marc21-drafts")
```
Comment (mb-wali) [inline invenio_records_marc21/api.py:25]: does marc21 need a draft record creation? @utnapischtim

File: invenio_records_marc21/cli.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+            <datafield tag='245' ind1='0' ind2='0'>\
+            <subfield code='a'>&lt;&lt;Die&gt;&gt; Internationale Werkbundsiedlung Wien 1932</subfield>\
+            <subfield code='c'>hrsg. von Josef Frank</subfield>\
+            </datafield>\
+            <datafield tag='264' ind1=' ' ind2='55'>\
+            <subfield code='a'>Wien</subfield>\
+            <subfield code='b'>Schroll</subfield>\
+            <subfield code='c'>1932</subfield>\
+            </datafield>\
+            </record>",
```
Comment (mb-wali) [inline invenio_records_marc21/cli.py:66]: use a separate file - ```**.json```

File: invenio_records_marc21/services/schemas/__init__.py [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+    #     """Loads the 'extensions' field.
+
+    #     :params value: content of the input's 'extensions' field
+    #     """
+    #     current_app_metadata_extensions = (
+    #         current_app.extensions['invenio-rdm-records'].metadata_extensions
+    #     )
+    #     ExtensionSchema = current_app_metadata_extensions.to_schema()
+
+    #     return ExtensionSchema().load(value)
```
Comment (mb-wali) [inline invenio_records_marc21/services/schemas/__init__.py:83]: What are these?

**Review by mb-wali — COMMENTED** [review]
why the need of adding vocabularies? if we wish to use the same vocabularies such as ```access_rights``` then why not use them from existing modules.

File: invenio_records_marc21/api.py (hunk 1 above)
Comment (philippgualdi) [reply invenio_records_marc21/api.py:25]: It sounds to me like a good idea to make a draft first. In the feature, the module is used to import vom campus online and send it further to alma where the record will be modified again and then it'll be published.

File: invenio_records_marc21/cli.py (hunk 2 above)
Comment (philippgualdi) [reply invenio_records_marc21/cli.py]: I moved the fake record to a separate file.

File: invenio_records_marc21/services/schemas/__init__.py (hunk 3 above)
Comment (philippgualdi) [reply invenio_records_marc21/services/schemas/__init__.py]: I removed these comments.

**Comment by philippgualdi** [comment]
> why the need of adding vocabularies? if we wish to use the same vocabularies such as `access_rights` then why not use them from existing modules.

Are there in the invenio-vocabularies module already added the access_rights?

**Comment by mb-wali** [comment]
> > why the need of adding vocabularies? if we wish to use the same vocabularies such as `access_rights` then why not use them from existing modules.
> 
> Are there in the invenio-vocabularies module already added the access_rights?

at the moment these vocabularies are inside [invenio-rdm-records](https://github.com/inveniosoftware/invenio-rdm-records/tree/master/invenio_rdm_records/vocabularies) but not sure if they planed to move them to invenio-vocabularies.

**Review by mb-wali — APPROVED** [review]

**Pull request merged**